tests/operators/test_bash_operator.py

```diff
@@ -98,3 +105,22 @@ def test_return_value(self):
         return_value = bash_operator.execute(context={})
 
         self.assertEqual(return_value, 'stdout')
+
+    def test_task_retries(self):
+        bash_operator = BashOperator(
+            bash_command='echo "stdout"',
+            task_id='test_task_retries',
+            retries=2,
+            dag=None
+        )
+
+        self.assertEqual(bash_operator.retries, 2)
+
+    def test_default_retries(self):
+        bash_operator = BashOperator(
+            bash_command='echo "stdout"',
+            task_id='test_default_retries',
+            dag=None
+        )
+
+        self.assertEqual(bash_operator.retries, 3)
```

Comment:
Does this test pass? Does default values are evaluated at module parse time, so changing the config value at runtime here won't affect the already-loaded default argument in the fn.

Reply:
yup, you are correct, it doesn't pass. any pointers to fix this?

Reply:
Either mess around with reloading modules (I think we do it in the logging tests?) or just don't test this?